### PR TITLE
Update log messages to use structured logs

### PR DESCRIPTION
**Pull request by JoelSpeed**
These logs messages were recently merged and weren't using structured logs, they should be

### CODE REVIEW

File: pkg/controllers/controlplanemachineset/watch_filters.go [hunk 1]
```diff
@@ -79,7 +84,11 @@ func filterControlPlaneMachineSet(namespace string) predicate.Predicate {
 		shouldReconcile := cpms.GetNamespace() == namespace && cpms.GetName() == clusterControlPlaneMachineSetName
 
 		if shouldReconcile {
-			klog.V(4).Infof("reconcile triggered by control plane machine set %s/%s", obj.GetNamespace(), obj.GetName())
+			klog.V(4).Infof(
```
Comment (RadekManak) [inline pkg/controllers/controlplanemachineset/watch_filters.go:87]: Should this be Info not Infof?

Comment (JoelSpeed) [reply pkg/controllers/controlplanemachineset/watch_filters.go:87]: Yes it should, good catch

**Comment by JoelSpeed** [comment]
/override ci/prow/e2e-aws-upgrade

The ovn and serial jobs both passed, and this is a logging change. I can see the CPMS running in the upgrade test history so I'm content that this PR isn't causing an issue

**Pull request merged**